### DISCUSSION TITLE
Executing Dynamic Code in a Reveal.js Presentation

### DISCUSSION THREAD

**Discussion by giscus** [cfjedimaster/raymondcamden2023#43] [Comments]
# Executing Dynamic Code in a Reveal.js Presentation

A little hack to add some dynamic-ness to my Reveal.js presentations

https://www.raymondcamden.com/2024/11/12/executing-dynamic-code-in-a-revealjs-presentation

**Comment by remotesynth** [comment]
When I was at LaunchDarkly, I built a reveal.js presentation that actually executed code to change the slide deck itself using feature flags. Flipping a flag would actually make the slide itself change. Total hackery but it was fun.

**Reply by cfjedimaster** [reply]
Sweet. Reveal *really* is darn flexible.
